test/test_extended_models.py

```diff
@@ -95,8 +95,8 @@ def test_schema_meta_validation(model_fn):
     # mandatory fields for each computer vision task
     classification_fields = {"categories", ("metrics", "acc@1"), ("metrics", "acc@5")}
     defaults = {
-        "all": {"metrics", "min_size", "num_params", "recipe"},
-        "models": classification_fields | {"_docs"},
+        "all": {"metrics", "min_size", "num_params", "recipe", "_docs"},
```

Comment:
Now that this is mandatory for all weights, we can change these lines

https://github.com/pytorch/vision/blob/ae1d70713a542aac905cf403844a21b38b2be593/docs/source/conf.py#L348-L350

into 

```py
 lines += [meta_with_metrics.pop("_docs")]   # Custom per-Weights docs 
```

Reply:
good spot done.